ui/ui-components/components/tabs/Group.tsx

```diff
@@ -3,11 +3,21 @@ import { Models } from "../../utils/apiData";
 import { grouparooUiEdition } from "../../utils/uiEdition";
 
 export default function GroupTabs({ group }: { group: Models.GroupType }) {
-  let tabs = ["edit", "rules"];
+  let tabs = [];
+  let defaultTab = "rules";
 
-  if (grouparooUiEdition() === "enterprise") {
-    tabs.push("members", "destinations", "runs", "logs");
+  switch (grouparooUiEdition()) {
+    case "community":
+      tabs = ["members"];
+      defaultTab = "members";
+      break;
```

Comment:
I think this means that community lost `edit` and `rules`, which it used to

Reply:
It didn't have any pages previously - this `GroupTabs` component was returning `edit` and `rules` by default because community wasn't using `GroupTabs` at all. If you'll notice the `/pages/model/[modelId]/group` dir for ui-community, it didn't exist before and `members` was just added now.